services/hanke-service/src/integrationTest/kotlin/fi/hel/haitaton/hanke/attachment/application/ApplicationAttachmentServiceITest.kt

```diff
@@ -0,0 +1,215 @@
+package fi.hel.haitaton.hanke.attachment.application
+
+import assertk.assertThat
+import assertk.assertions.each
+import assertk.assertions.endsWith
+import assertk.assertions.hasSize
+import assertk.assertions.isEqualTo
+import assertk.assertions.isNotNull
+import fi.hel.haitaton.hanke.DatabaseTest
+import fi.hel.haitaton.hanke.HankeEntity
+import fi.hel.haitaton.hanke.HankeRepository
+import fi.hel.haitaton.hanke.application.ApplicationEntity
+import fi.hel.haitaton.hanke.application.ApplicationNotFoundException
+import fi.hel.haitaton.hanke.attachment.APPLICATION_ID
+import fi.hel.haitaton.hanke.attachment.FILE_NAME_PDF
+import fi.hel.haitaton.hanke.attachment.HANKE_TUNNUS
+import fi.hel.haitaton.hanke.attachment.USERNAME
+import fi.hel.haitaton.hanke.attachment.common.ApplicationAttachmentRepository
+import fi.hel.haitaton.hanke.attachment.common.ApplicationAttachmentType
+import fi.hel.haitaton.hanke.attachment.common.ApplicationAttachmentType.MUU
+import fi.hel.haitaton.hanke.attachment.common.ApplicationAttachmentType.VALTAKIRJA
+import fi.hel.haitaton.hanke.attachment.common.AttachmentNotFoundException
+import fi.hel.haitaton.hanke.attachment.common.AttachmentScanStatus
+import fi.hel.haitaton.hanke.attachment.common.AttachmentScanStatus.OK
+import fi.hel.haitaton.hanke.attachment.common.AttachmentUploadException
+import fi.hel.haitaton.hanke.attachment.testFile
+import fi.hel.haitaton.hanke.factory.AlluDataFactory
+import java.util.*
```

Comment:
Please don't add wildcard imports.

Reply:
I've disabled wildcard imports from the IDE, I do not know why it keeps doing this... I'll fix this.

Reply:
Fixed.

Reply:
I think IDEA doesn't expand wildcard imports if they're already present.